mm2src/coins/utxo/utxo_wrapper_block_header_storage.rs

```diff
@@ -0,0 +1,73 @@
+use crate::utxo::rpc_clients::ElectrumBlockHeader;
+use crate::utxo::utxo_block_header_storage::{BlockHeaderStorage, BlockHeaderStorageError, BlockHeaderStorageOps,
+                                             InitBlockHeaderStorageOps};
+#[cfg(target_arch = "wasm32")]
+use crate::utxo::utxo_indexedb_block_header_storage::IndexedDBBlockHeadersStorage;
+#[cfg(not(target_arch = "wasm32"))]
+use crate::utxo::utxo_sql_block_header_storage::SqliteBlockHeadersStorage;
+use async_trait::async_trait;
+use chain::BlockHeader;
+use common::mm_ctx::MmArc;
+use common::mm_error::MmError;
+use std::collections::HashMap;
+
+impl InitBlockHeaderStorageOps for BlockHeaderStorage {
```

Comment:
Why these traits are implemented in a separate file? I guess there was a structure like `UtxoBlockHeaderStorage` before?